### ISSUE TITLE
unable to download

### ISSUE THREAD

**Issue by Thorazin** [ocristopfer/cockpit-sensors-old#38]
I get a 404 error when trying to download for install.

**Comment by wikijm** [comment]
Hi @Thorazin 

Files cockpit-sensors.tar.xz and cockpit-sensors.deb are missing on release v1.4.2:
![image](https://github.com/ocristopfer/cockpit-sensors/assets/10724809/b67d54fc-672d-43cd-b568-4fee149e254d)

You can use commands below instead:
```bash
cd /tmp
wget https://github.com/ocristopfer/cockpit-sensors/archive/refs/tags/v1.4.2.tar.gz
tar -xzf v1.4.2.tar.gz
apt install lm-sensors #then apt --fix-broken install if something is missing
dpkg -i ./cockpit-sensors-1.4.2/cockpit-sensors.deb
rm -fR ./*1.4.2*
```

**Comment by Thorazin** [comment]
Thank you.

**Comment by quux-aivar** [comment]
Does not work

**Comment by ocristopfer** [comment]
fix o new release 1.4.2
